tests/PHPStan/Analyser/AnalyserIntegrationTest.php

```diff
@@ -370,8 +371,7 @@ public function testBug4288(): void
 		$class = $reflectionProvider->getClass(MyClass::class);
 		$parameter = ParametersAcceptorSelector::selectSingle($class->getNativeMethod('paginate')->getVariants())->getParameters()[0];
 		$defaultValue = $parameter->getDefaultValue();
-		$this->assertInstanceOf(ConstantIntegerType::class, $defaultValue);
-		$this->assertSame(10, $defaultValue->getValue());
+		$this->assertInstanceOf(MixedType::class, $defaultValue);
```

Comment:
this default value is now generalized, since it is referenced from within a trait via class-constant